src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/transform/action/get/GetTransformResponse.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.get
+
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.model.Transform
+import org.elasticsearch.action.ActionResponse
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.common.xcontent.ToXContentObject
+import org.elasticsearch.common.xcontent.XContentBuilder
+import org.elasticsearch.rest.RestStatus
+import java.io.IOException
+
+class GetTransformResponse : ActionResponse, ToXContentObject {
+    var id: String
+    var version: Long
+    var seqNo: Long
+    var primaryTerm: Long
+    val status: RestStatus
+    val transform: Transform?
+
+    constructor(
+        id: String,
+        version: Long,
+        seqNo: Long,
+        primaryTerm: Long,
+        status: RestStatus,
+        transform: Transform?
+    ) : super() {
+        this.id = id
+        this.version = version
+        this.seqNo = seqNo
+        this.primaryTerm = primaryTerm
+        this.status = status
+        this.transform = transform
+    }
```

Comment:
since this constructor is not doing anything special you can just do this instead - its more concise and achieving the same

```
class GetTransformResponse(
    val id: String,
    val version: Long,
    val seqNo: Long,
    val primaryTerm: Long,
    val status: RestStatus,
    val transform: Transform?
) : ActionResponse(), ToXContentObject {
```